common/resource/resource.go

```diff
@@ -61,6 +61,7 @@ type (
 		GetArchivalMetadata() archiver.ArchivalMetadata
 		GetClusterMetadata() cluster.Metadata
 		GetSearchAttributesProvider() searchattribute.Provider
+		GetSearchAttributesManager() searchattribute.Manager
```

Comment:
what is the difference?

Reply:
add this to a comment somewhere visible?

Reply:
I think it is obvious if you check definition of both interfaces.